docker/install-dynamic-plugins.py

```diff
@@ -0,0 +1,118 @@
+import os
+import sys
+import yaml
+import tarfile
+import shutil
+import subprocess
+
+dynamicPluginsRoot = sys.argv[1]
+maxEntrySize = os.environ.get('MAX_ENTRY_SIZE', 10000000)
+
+dynamicPluginsFile = os.path.join(dynamicPluginsRoot, 'dynamic-plugins.yaml')
+dynamicPluginsGlobalConfigFile = os.path.join(dynamicPluginsRoot, 'app-config.dynamic-plugins.yaml')
+
+# test if file dynamic-plugins.yaml exists
+if not os.path.isfile(dynamicPluginsFile):
+    print(f'No {dynamicPluginsFile} file found. Skipping dynamic plugins installation.')
+    with open(dynamicPluginsGlobalConfigFile, 'w') as file:
+        file.write('')
+        file.close()
+    exit(0)
+
+with open(dynamicPluginsFile, 'r') as file:
+  plugins = yaml.safe_load(file)
+
+if plugins == '' or plugins is None:
+    print(f'{dynamicPluginsFile} file is empty. Skipping dynamic plugins installation.')
+    with open(dynamicPluginsGlobalConfigFile, 'w') as file:
+        file.write('')
+        file.close()
+    exit(0)
+
+class InstallException(Exception):
+    """Exception class from which every exception in this library will derive."""
+    pass
+
+# test that plugins is a list
+if not isinstance(plugins, list):
+    raise InstallException(f'{dynamicPluginsFile} content must be a list')
+
+globalConfig = {
+  'dynamicPlugins': {
+        'rootDirectory': 'dynamic-plugins-root'
```

Comment:
```suggestion
        'rootDirectory': 'dynamic-plugins'
```

The dockerfile uses `dynamic-plugins`

Reply:
No, that's not the same.
`dynamic-plugins` is the source folder for dynamic plugins that will be built inside the docker image
`dynamic-plugins-root` is the destination folder in which all the dynamic plugins packages (not the sources) will be unpacked.